### PR TITLE
Encryption key changes

### PR DESCRIPTION
**Pull request by Afya212**
## About The Pull Request

Adds the ability for med, eng, sci, sup, serv encryption keys to be able to be made from department techfabs when researched. Adds ability to order encryption keys from cargo. Removed redundant techfab circuit from head lockers, departments already start with techfabs, go to science for more.
Any feedback on cargo pricing/techfab construction cost balancing is appriciated.

## Why It's Good For The Game

QoL stuff, able to get more encryption keys if needed.

## Changelog
:cl: Afya
add: Med, eng, sci, sup, and serv encryption keys now researchable/craftable
add: Cargo can now order encryption keys
tweak: looc assumes less
/:cl:

### CODE REVIEW

File: modular_skyrat/code/modules/cargo/packs/misc.dm [hunk 1]
```diff
@@ -0,0 +1,81 @@
+
+//Reminders-
+// If you add something to this list, please group it by type and sort it alphabetically instead of just jamming it in like an animal
+// cost = 700- Minimum cost, or infinite points are possible.
+
+//////////////////////////////////////////////////////////////////////////////
+////////////////////////// Headset Encyption Keys ////////////////////////////
+//////////////////////////////////////////////////////////////////////////////
+
+/datum/supply_pack/misc/encryption/bin_key
```
Comment (Akrilla) [inline modular_skyrat/code/modules/cargo/packs/misc.dm:10]: Probably access captain on this too

Comment (Afya212) [reply modular_skyrat/code/modules/cargo/packs/misc.dm:10]: It's only able to be ordered via emagging, and if you're buying something from the black market you probably want to be able to access it without a hassle/lock.

Comment (Akrilla) [reply modular_skyrat/code/modules/cargo/packs/misc.dm:10]: mb didn't see hidden

**Comment by ForrestWick** [comment]
Those techfab boards are literally irreplaceable, meaning if someone steals the roundstart machine board and the one from a head locker, you're fucked

**Comment by Afya212** [comment]
Could have sworn you could make them, I'll put it back for now.

**Pull request merged**